### DISCUSSION TITLE
Allow toggling the grid of a ui.scene

### DISCUSSION THREAD

**Discussion by falkoschindler** [zauberzeug/nicegui#1261] [Ideas / Feature Requests]
As @WolfgangFahl suggested in #1246, it would be nice to support toggling the visibility of the ground plane and/or the grid of a UI scene. Once we are at it, we might as well add optional coordinate axes like Wolfgang did in #1251.

**Comment by WolfgangFahl** [comment]
any progress?

**Reply by falkoschindler** [reply]
Not yet. So far this feature request hasn't seen any upvotes (apart from the two of us), let alone contributions.

Anyway, we could certainly introduce a method like `set_grid_visibility`. But playing around with this approach I noticed that there's also the ground plane which should be toggled as well - or shouldn't it? Sometimes you might want to hide both, sometimes only the grid. So we'd need even more methods for showing/hiding the grid, the ground plane, and maybe a coordinate axes indicator in the future. This makes the API less trivial.

**Comment by WolfgangFahl** [comment]
the solution in nicegui_widgets works by now see e.g. nicescad.bitplan.com
